digital_asset_types/src/dao/mod.rs

```diff
@@ -198,6 +192,23 @@ impl SearchAssetsQuery {
             joins.push(rel);
         }
 
+        if let Some(cv) = self.creator_verified {
+            conditions = conditions.add(asset_creators::Column::Verified.eq(cv));
```

Comment:
Something to consider is that without specifying the creators themselves, there is no index being hit. That means in some scenarios this query could be very slow. But I think those would only happen in rare scenarios.